test/built-ins/String/prototype/normalize/return-abrupt-from-form.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.12
+description: >
+  Returns abrupt from ToString(form)
```

Comment:
I'm ok to merge, but that should affect the other open PRs and we could also track the others already merged.